workflow/runtime/suite.yaml

```diff
@@ -39,31 +42,41 @@ cycled_suite_alarms: &cycled_suite_alarms
 
   # gfs - cycles for which the gfs is run
   gfs: !FirstTrue
-    - when: !calc doc.settings.gfs_cyc == 0
+    # No free forecast
+    - when: !calc doc.settings.STEP_GFS == doc.zero_dt
       do: !Clock  # required but ignored because gfs is disabled
         start: !calc suite.Clock.start
         end: !calc suite.Clock.end
         step: !calc suite.Clock.step
-    - when: !calc doc.settings.gfs_cyc == 1
-      do: !calc suite.Clock.for_alarm(doc.gfs_clock_1)
-    - when: !calc doc.settings.gfs_cyc == 2
-      do: !calc suite.Clock.for_alarm(doc.gfs_clock_2)
-    - when: !calc doc.settings.gfs_cyc == 4
-      do: !calc suite.Clock.for_alarm(doc.gfs_clock_4)
-    - otherwise: !error Unknown gfs clock settings {doc.settings.gfs_cyc}
+    - otherwise: !Clock
+        start: !FirstTrue
+          - when: !calc doc.settings.STEP_DA == doc.zero_dt
+            # No DA
+            do: !calc suite.Clock.start
+            # Run one DA-only cycle before starting GFS
+          - otherwise: !calc suite.Clock.start+suite.Clock.step
+        end: !calc suite.Clock.end
+        step: !calc doc.settings.STEP_GFS
 
-  gfs_00_12: !calc suite.Clock.for_alarm(doc.gfs_clock_2)
+  gfs_00_12: !FirstTrue
+    - when: !calc doc.settings.STEP_GFS >= doc.day_dt
+      do: !calc suite.Clock.for_alarm(doc.gfs_clock_1)
+    - otherwise: !calc suite.Clock.for_alarm(doc.gfs_clock_2)
 
 
 # suite_defaults - this is inherited by the actual suite definitions
 # It sets defaults for various variables that are generally not overridden
 suite_defaults: &suite_defaults
   # Clock - the list of cycles to run
   Clock: !Clock
-    # NEED TO FIX FOR LESS FREQUENT GFS
     start: !calc doc.settings.SDATE
     end: !calc doc.settings.EDATE
-    step: !timedelta "6:00:00"
+    step: !FirstTrue
+      - when: !calc doc.settings.STEP_DA == doc.zero_dt
```

Comment:
Is this where the default for STEP_DA is set?

Reply:
Default is !timedelta 06:00:00

Reply:
No, that is set in defaults/settings.yaml. This is determining the master clock for the experiment. Since DA can happen more often than the forecast, STEP_DA is used unless it is off (zero).